playbook/app/pb_kits/playbook/pb_badge/badge.test.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { cleanup, render, screen } from '../utilities/test-utils'
+
+import Badge from './_badge'
+
+const testId = "badge"
+
+test('default class name', () => {
+  render(
+    <Badge
+        data={{ testid: testId }}
+        text="+1"
+    />
+  )
+
+  const kit = screen.getByTestId(testId)
+
+  expect(kit).toHaveClass('pb_badge_kit_neutral')
+})
+
+test('primary class name', () => {
+  render(
+    <Badge
+        data={{ testid: testId }}
+        text="+1"
+        variant="primary"
+    />
+  )
+
+  const kit = screen.getByTestId(testId)
+
+  expect(kit).toHaveClass('pb_badge_kit_primary')
+})
+
+test('displays text content', () => {
+  render(
+    <Badge
+        text="+1"
+        variant="primary"
+    />
+  )
+
+  const text = screen.getByText("+1")
+  expect(text).toBeInTheDocument()
+})
+
+test('displays rounded corners', () => {
+  render(
+    <Badge
+        data={{ testid: testId }}
+        rounded
+        text="+1"
+        variant="primary"
+    />
+  )
+
+  const kit = screen.getByTestId(testId)
+  expect(kit).toHaveClass('pb_badge_kit_primary_rounded')
+})
+
+test('displays color variants', () => {
+  [
+    "success",
+    "warning",
+    "error",
+    "info"
+  ].forEach((colorVariant) => {
+    render(
+      <Badge
+          data={{ testid: testId }}
+          text={colorVariant}
+          variant={colorVariant}
+      />
+    )
+    const kit = screen.getByTestId(testId)
+    expect(kit).toHaveClass(`pb_badge_kit_${colorVariant}`)
+
+    cleanup()
```

Comment:
For this `cleanup()` call, the screen should be the target of the tear-down/garbage collection along with any variable stores.